components/atoms/Select/single-select.tsx

```diff
@@ -27,12 +63,45 @@ const SingleSelect = ({ placeholder, value, onValueChange, options, position }:
         <SelectValue className="w-10 truncate" placeholder={placeholder ?? "Select time zone"} />
       </SelectTrigger>
 
-      <SelectContent position={position ?? "item-aligned"} className="z-50 bg-white">
-        {options.map((option, index) => (
-          <SelectItem title={option.label} key={`timezone_${index}`} value={option.value}>
-            {truncateString(option.label, 30)}
-          </SelectItem>
-        ))}
+      <SelectContent position={position ?? "item-aligned"} className="z-50 relative bg-white">
```

Comment:
Curious if the z-index is necessary. I know it was here previously.

Reply:
It's needed to keep the items above other elements when open